index.jsx

```diff
@@ -322,6 +323,17 @@ const setGit = repo => {
     });
 };
 
+const getEditor = editorType => {
+    switch (editorType) {
+        case 'vscode':
+            return '/usr/local/bin/code';
+        case 'atom':
+            return '/usr/local/bin/atom';
+        default:
+            return 'open';
```

Comment:
Should this be a default? IIRC `open` isn’t available on Linux.

Reply:
Hmm I'm not really sure how to determine what the default action should be.
In that case I also don't think /usr/local/bin would work on Windows & Linux

Reply:
If you merge the two options as suggested above, you could throw if there’s an invalid truthy value provided.